crates/sound/src/vhu_sound.rs

```diff
@@ -131,7 +141,7 @@ impl VhostUserSoundThread {
     }
 
     /// Process the messages in the vring and dispatch replies
-    fn process_control(&self, vring: &VringRwLock) -> Result<bool> {
+    fn process_control(&self, vring: &VringRwLock, stream_info: &Vec<StreamInfo>) -> Result<bool> {
```

Comment:
```suggestion
    fn process_control(&self, vring: &VringRwLock, stream_info: &[StreamInfo]) -> Result<bool> {
```
`cargo clippy` should normally complain about this